### PR TITLE
Update kicad-zh_CN_v5.1.1.008

### PR DESCRIPTION
**Pull request by taotieren**
## zh_CN
- 5606 translated messages
- 0 fuzzy translations
- 0 untranslated messages

### CODE REVIEW

File: zh_CN/kicad.po [hunk 1]
```diff
@@ -11252,11 +11237,11 @@ msgid "Save Workbook"
 msgstr "保存工作簿"
 
 #: eeschema/sim/sim_plot_frame_base.cpp:35
-msgid "Save as Image"
+msgid "Save as image"
```
Comment (nickoe) [inline zh_CN/kicad.po]: What is the reasoning to change the capitalization of the word image?

**Comment by nickoe** [comment]
@taotieren Please follow up on my comments. It seems that we are gonna tag 5.1.2 soon.

File: zh_CN/kicad.po [hunk 2]
```diff
@@ -10403,11 +10388,11 @@ msgstr "电源输出"
 
 #: eeschema/pin_type.cpp:62
 msgid "Open collector"
-msgstr "集电极开路"
+msgstr "Open collector"
 
 #: eeschema/pin_type.cpp:65
 msgid "Open emitter"
-msgstr "发射极开路"
+msgstr "Open emitter"
```
[the remark this answers is not in the document]
Comment (taotieren) [reply zh_CN/kicad.po]: I use poedit for translation, but poedit remembers some incorrect translations, not even for translation. I don't know if there's a problem with poedit. I'll check it carefully later. At the same time, I want to know how kicad. pot can be incrementally updated to zh_CN/kicad. po.

File: zh_CN/kicad.po (hunk 1 above)
Comment (taotieren) [reply zh_CN/kicad.po]: This kicad.po file was reproduced from the kicad-5.1.1 source code. Is the source code changed?

**Comment by taotieren** [comment]
I want to know how kicad. pot can be incrementally updated to zh_CN/kicad. po.In this way, I only need to care about adding new translations, rather than re-calibrating them every time they are updated.

**Comment by taotieren** [comment]
@nickoe  If the source code is updated, please update the branch corresponding to i18n. In this way, the translator only cares about the translation problem, instead of having to get the latest kicad. pot from the source code to regenerate the new kicad. po.

**Pull request merged**